python/requirements-test.txt

```diff
@@ -1,5 +1,6 @@
 cffi
 hypothesis
+packaging
```

Comment:
You should also update `requirements-wheel-test.txt`

Reply:
Good catch, I'll add it.